libraries/AP_ExternalAHRS/AP_ExternalAHRS_LORD.cpp

```diff
@@ -440,6 +440,12 @@ int8_t AP_ExternalAHRS_LORD::get_port(void) const
     return port_num;
 };
 
+// Get model/type name
+const char* AP_ExternalAHRS_LORD::get_name() const
+{
+    return "LORD";
```

Comment:
I did not look at the LORD code in detail, possibly we could do better than this.